WordPressKitTests/Mock Data/site-segments-single.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
It's unlikely that the service will return one result, but I wonder - would a payload for such a case include _just_ a dictionary, or would it be an array containing a single dictionary? 

The latter case evinces failures in `SiteSegmentsResponseDecodingTests`.

Reply:
On second thought, I don't think that concern impacts this test, as I see you're using it to decode a single `SiteSegment` instance.